draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -236,19 +277,19 @@ The document is divided into the following sections:
 
 # Optical Pluggable in a Device with Packet Functions
 
-{{figure-details-packet-optical-device}} shows a packet device from vendor X, which is connected to optical device, equipped with optical pluggables from vendor X and Y. This figure exposes the following internal and external interfaces:
+{{figure-details-packet-optical-device}} shows a host packet device from vendor X, which is connected to optical device, equipped with optical pluggables from vendor X and Y. This figure exposes the following internal and external interfaces:
 
-1. This interface provides the control of packet device packet and optical functions. It provides these functions which can be decoupled such that there is no overlap between the optical and packet control functions.
+1. This interface provides the control of host packet and optical functions. It provides these functions which can be decoupled such that there is no overlap between the optical and packet control functions.
 
 2. The optical controller has read/write access to the optical network
 
-3. The CMIS (content management interoperability services) communication interface between hosts (e.g., packet devices) and optical pluggable
+3. The CMIS (content management interoperability services) communication interface between host packet devices) and optical pluggables
 
 4. The data flow between the coherent pluggable and the packet data function through this interface. This is electrical interface between coherent pluggable and the host. {{building-blocks}} will discuss this in more details.
 
 5. Optical fiber connecting the optical devices to optical pluggables. This carries the flow of photonic signal from the optical device to the coherent pluggables. {{building-blocks}} will discuss this in more details.
 
-This draft presents the modeling of the coherent pluggable such as #1 and #2 in {{figure-details-packet-optical-device}} within a host packet device. The model presented in {{data-model}} consolidates properties of coherent pluggable on interfaces (4) and (5) where interface (5) provides the photonic/optical attributes and interface provides the host/electrical attributes.
+This draft presents the modeling of the coherent pluggable such as (4) and (5) in {{figure-details-packet-optical-device}} within a host packet device. The model presented in {{data-model}} consolidates properties of coherent pluggable on interfaces (4) and (5) where interface (5) provides the photonic/optical attributes and interface provides the host/electrical attributes.
```

Comment:
Nothing changed , you need to add interface 4 !!!!

Reply:
Can you suggest what is needed to be replaced/removed? I will revise the text

Reply:
"The model presented in {{data-model}} consolidates properties of coherent pluggable on interfaces (4) and (5) where interface (5) provides the photonic/optical attributes and interface provides the host/electrical attributes."
You need to add "....and interface (4) provides the host/electrical attributes"

Reply:
I have replaced the original text with your suggested one.